### ISSUE TITLE
Pre-Usability Review [IRIS] -- Accessibility Feedback

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#16646]
## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

### Must
* https://staging.va.gov/ask-a-question/veteran-information email address must be validated a second time. The potential for PII or sensitive information being sent to the wrong person is too high not to include the dual validation routines of entering a value, and ensuring it matches the value entered a second time.
* Date of Birth is inserted between the Branch of Service select and service start/end dates. This must be moved to another location, maybe right before or right after Social Security Number.

### Should
* Test with a screen reader or screen magnifier user as part of the usability sessions. Trevor can help you prepare for a remote screen reader session over Zoom if you'd like.

### Consider
* Review the design style for the question on Step 4. Specifically, the "Please enter your message below" answer could get long because that data comes from a `textarea`. The right-aligned bold text is not the easiest to read. If this is a standard pattern, please disregard. Otherwise this could be styled to be left-aligned and normal weight.

### Questions
* What is your recruiting pool? Are you requesting Perigean recruit any users with:
  * Cognitive issues (TBI, PTSD, ADHD, etc.)
  * Disability rating
  * Person of color
  * Assistive technology
* Are you planning to test with one or more mobile users? What about mobile screen readers?
* Is a chatbot still a possibility?

**Comment by murrayra** [comment]
Hi @1Copenut !

Just an update on the feedback - thank you for this! A few comments below:

**Must**

- _https://staging.va.gov/ask-a-question/veteran-information email address must be validated a second time. The potential for PII or sensitive information being sent to the wrong person is too high not to include the dual validation routines of entering a value, and ensuring it matches the value entered a second time._

I'm not sure where this was happening, but the devs are double checking that the validating is happening.  It is working here ([](https://staging.va.gov/ask-a-question/contact-information)) which is the current version of the form.

- _Date of Birth is inserted between the Branch of Service select and service start/end dates. This must be moved to another location, maybe right before or right after Social Security Number._

We'll incorporate that as part of our card [212](https://github.com/department-of-veterans-affairs/orchid/issues/212) and the devs will have this done in our current sprint before Dec. 15th, likely done by end of this week - let me know if this is ok to mark as done.

**Should**

- _Test with a screen reader or screen magnifier user as part of the usability sessions. Trevor can help you prepare for a remote screen reader session over Zoom if you'd like._

We'll follow up on doing this depending on who is recruited.

**Consider**

- _Review the design style for the question on Step 4. Specifically, the "Please enter your message below" answer could get long because that data comes from a textarea. The right-aligned bold text is not the easiest to read. If this is a standard pattern, please disregard. Otherwise this could be styled to be left-aligned and normal weight._

This is a standard pattern within the VA Design System.

**Questions**

- _What is your recruiting pool? Are you requesting Perigean recruit any users with:

Cognitive issues (TBI, PTSD, ADHD, etc.)
Disability rating
Person of color
Assistive technology

We're hoping to get any users in any of the above categories, and in particular cognitive challenges.  We also anticipate that the follow up usability testing we'll conduct will be more valuable in particular for the folks with cognitive, disability and assistive technology.  In the future (say Jan.?) we'll have the full list of topic drop downs from the VA lines of business, and will be able to see how that experience is and may also have things like the FAQs etc. and supportive text also built by then into the app.

- _Are you planning to test with one or more mobile users? What about mobile screen readers?_

Yes, we should be testing with mobile, and will look into mobile screen readers as well.

- _Is a chatbot still a possibility?_

We're still in discussion about this with various folks at the VA as a part of a larger rethinking of how folks Get Help in the VA so stay tuned  #comingattractions

**Comment by 1Copenut** [comment]
Thank you @murrayra for the comments and follow up. I'll move this item to closed.
